src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -187,6 +187,14 @@ public void deleteOldJobs() {
     deleteJobs(jobsToDelete);
   }
 
+  @Transactional
```

Comment:
Lets remove @Transaction, it does not necessary here, and also previosly we had issue: https://issues.folio.org/browse/MODEXPS-91
'ERROR JobUpdatesService    Update for unknown job d151339a-b964-482b-aee0-50908a226a3d.'
when using transactional method upsertAndSaveToKafka() that saves data but when quering the same job by id for sending to kafka - fetched Not found exception because transaction has not been yet committed.

Reply:
@SerhiiNosko :-
if we try to call @transactional method from non-transactional method then sonar is introducing a bug as below.
**@transactional requirement is incompatible with the one for this method**